benchmarking/smartseq2/merge_picard_metrics/merge_picard_mets.py

```diff
@@ -0,0 +1,73 @@
+from crimson import picard
+import pandas as pd
+import numpy as np
+from google.cloud import storage
+import json
+from os.path import basename
+import sys
+import requests
+import argparse
+
+def merge_picard_metrics(uuid,met_name,output_name):
+    """
+    piepline output picard QC metrics at sinle cell/sample level.
+    This functin is called to merge/aggregate QC metrics by metrics type and then merge multiple QC measurement 
+    into single matrix file. In this file, column is sample/cell and row is QC metrics
+    param uuid: run uuid
+    param met_name: metrics name with workflow name and subworkflow name as prefix. such as ['run_pipelines.RunStarPipeline.alignment_summary_metrics']
+    param output_name: output name, csv file
+    """
+    # set up auth
+    client = storage.Client()
```

Comment:
Here's a great internal reason to do the refactoring I suggested above: you could very likely re-use some of the functions I suggested: 

```
parse_sample_uuid()
merge_expression_vectors()  # maybe this could be could be general to "merge data"
parse_metrics()  # this is new in this function and needs to be created from your below code. 
write_output()